docs/content/concepts/repositories-workspaces/workspaces.mdx

```diff
@@ -1,201 +1,281 @@
 ---
-title: Workspaces | Dagster
-description: Dagster Workspaces are collections of user-defined repositories and information about where they reside.
+title: Workspaces | Dagster Docs
+description: Dagster Workspaces are collections of user-defined code locations and information about where they reside.
 ---
 
 # Workspaces
 
-A _workspace_ is a collection of user-defined [repositories](/concepts/repositories-workspaces/repositories) and information about where to find them. Dagster tools, like Dagit and the Dagster CLI, use workspaces to load user code.
+A workspace is a collection of user-defined [code locations](/concepts/code-locations) and information about where to find them. Workspaces can accept code locations defined using a <PyObject object="Definitions" /> object or a <PyObject object="repository" decorator /> decorator.
 
-## Relevant APIs
-
-| Name                                       | Description                                                                                                   |
-| ------------------------------------------ | ------------------------------------------------------------------------------------------------------------- |
-| <PyObject object="repository" decorator /> | The decorator used to define repositories. The decorator returns a <PyObject object="RepositoryDefinition" /> |
-
-## Overview
-
-A Dagster Workspace can contain multiple [repositories](/concepts/repositories-workspaces/repositories) sourced from a variety of different locations, such as Python modules or Python virtualenvs.
+Prior to the introduction of <PyObject object="Definitions" />, workspaces were used to enable Dagster tools to load user code in tools like Dagit or the CLI. In a `Definitions` world, workspaces are only required to load multiple code locations as part of an [open source (OSS) deployment](/deployment). For local development, `Definitions` users can utilize a [`pyproject.toml` file](/concepts/code-locations).
```

Comment:
Prior to the introduction of `<PyObject object="Definitions" />`, workspace files were used to enable Dagster tools to load user code in tools like Dagit or the CLI. In a `Definitions` world, workspace files are only required to load multiple code locations in customized environments, for example as as part of an [open source (OSS) deployment](/deployment).

For local development within a single python environment, `Definitions` users can use the `-m` or `-f` flags with our CLI tools, and set [`pyproject.toml` file](/concepts/code-locations) to avoid using command line flags entirely.